scripts/setDefaultIndex.py

```diff
@@ -141,6 +148,31 @@ def main():
     else:
         logging.info("No missing index-pattern fields.")
 
+    logging.info("================================== EVENTS INDEX PATTERN ==================================")
+
+    index_pattern_doc_created = create_document_if_it_doesnt_exist(esUtil.KIBANA_INDEX,
```

Comment:
You need to change the variable names of all these in the EVENTS INDEX PATTERN section so that they are unique from the previous section.